ppr-ui/src/assets/styles/base.scss

```diff
@@ -398,6 +398,16 @@ ul {
   }
 }
 
+// Dynamic Counters for stacked sections
+.increment-sections {
+  counter-reset: section 0
+}
+
+.increment-sections section h2:before {
```

Comment:
It's a nice CSS trick, but I'm not a big fan, especially when sometimes the design asks for a manual section override. Also, I am not sure how this would work for accessibility. Time will tel, but for now 👍🏻